couchbase-operator/templates/sync-gateway-deployment.yaml

```diff
@@ -30,6 +48,25 @@ spec:
         env:
         - name: GOMAXPROCS
           value: "1"
+        ports:
+          - containerPort: 4984
+            name: public
+          {{- if .Values.syncGateway.admin.enabled }}
+          - containerPort: 4985
+            name: admin
+          {{- end }}
+        resources:          
+{{ toYaml .Values.syncGateway.resources | indent 10 }}
+      {{- if .Values.syncGateway.monitoring.prometheus.enabled }}
+      - name: exporter
+        image: {{ default "couchbasesamples/sync-gateway-prometheus-exporter:latest" .Values.syncGateway.monitoring.prometheus.image }}
```

Comment:
Can you remove the default image string here since it's already set as default in values file.
I don't want to maintain values here.
The user would have to actively remove the string and if they get a failure they will know why.

Reply:
refactored this to use the same convention as the SyncGW image configuration and removed default from manifest:

```
monitoring:
    prometheus:
      # defines whether Prometheus metric collection is enabled
      enabled: false
      # image used by the Sync Gateway to perform metric collection
      # (injected as a "sidecar" in each Sync Gateway Pod)
      image:
        repository: couchbasesamples/sync-gateway-prometheus-exporter
        tag: latest
```